rules/have-a-definition-of-ready/rule.md

```diff
@@ -27,31 +28,42 @@ guid: 38162fd3-f74a-4c7d-908d-59ed622965b9
 
 <!--endintro-->
 
-Just like how the team has a [Definition of Done](/definition-of-done) as a checklist for completing a PBI, the Product Owner needs a Definition of Ready to get the PBI in a state that’s ready for it to be added to a Sprint. This should be done as part of the refinement (aka grooming) process and will greatly streamline the Sprint Planning meeting.
+Just like how the team has a **[Definition of Done](/definition-of-done)** as a checklist for completing a PBI; the Product Owner needs a **Definition of Ready (DoR)** to get the PBI in a state that’s ready to be added to a Sprint. This should be done as part of the refinement (aka grooming) process and will greatly streamline the Sprint Planning meeting.
 
-A recommended “Definition of Ready” would be:
+A recommended “Definition of Ready” would have:
 
-* Has enough detail for the team to action (usually via good [Acceptance Criteria](/acceptance-criteria))
-* Has [Business Value](/do-you-estimate-business-value) assigned
-* Has effort assigned
-* Is in the Approved state
+* Enough detail for the team to action (usually via good **[Acceptance Criteria](/acceptance-criteria)**)
+* **[Business Value](/do-you-estimate-business-value)** assigned
+* **Effort** assigned
+* **Approved** state
 
 ::: good
 ![Figure: PBI that meet the DoR](PBI-with-DOR.png)
 :::
 
-After the PBI is approved ask for a signature (or simply an initial) as a prove of approval. Check out this rules [ask-clients-approval](/do-you-estimate-business-value)
+When the PBI is approved, [ask for a signature (or simply an initial) as a prove of approval](/ask-clients-approval).
 
-**Notes:**
+::: info
+**Note:** PBIs can still be accepted into a Sprint even if they do **not** meet the DoR. When that happens, there will be more risk and uncertainty around the PBI, but this should not stop the team from doing the work. The recommendation is that PBI's are ready, but this is not mandatory.
```

Comment:
I think in practice this usually indicates a poorly functioning team.

I recommend adding a warning to discourage it.

If we look at what the Scrum Guide says
"Product Backlog items that can be Done by the Scrum Team within one Sprint are deemed ready for selection in a Sprint Planning event. "

The argument I would make is that a PBI cannot be evaluated properly for being done in one Sprint unless it has enough detail.